src/config/artifacts.rs

```diff
@@ -0,0 +1,183 @@
+use crate::config::Config;
+use crate::errors::*;
+use crate::site::html::build_common_html;
+use axohtml::elements::{div, script, span};
+use axohtml::{html, text, unsafe_text};
+use cargo_dist_schema::{ArtifactKind, DistManifest};
+use serde::Deserialize;
+use std::fs::File;
+use std::io::Write;
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum ArtifactSystem {
```

Comment:
i feel like this could be its own crate (don't need to do right now, but just noting it)

Reply:
yeah i agree with that, curious what you think @Gankra